src/components/Modals/ProjectCardModal/ProjectCardModal.tsx

```diff
@@ -0,0 +1,198 @@
+                    </label>
+                    <input
+                      className="w-full px-3 py-2 border border-gray-300 rounded"
+                      type="text"
+                      name="question1"
+                      value={answerOne}
+                      onChange={(e) => setAnswerOne(e.target.value)}
+                    />
+                  </div>
+                  <div>
+                    <label className="block text-gray-700" htmlFor="question2">
+                      Question 2
+                    </label>
+                    <input
+                      className="w-full px-3 py-2 border border-gray-300 rounded"
+                      type="text"
+                      name="question2"
+                      value={answerTwo}
+                      onChange={(e) => setAnswerTwo(e.target.value)}
+                    />
+                  </div>
+                  <div className="flex justify-end space-x-4">
+                    <button
+                      className="px-4 py-2 bg-gray-200 text-gray-700 rounded hover:bg-gray-300"
+                      type="button"
+                      onClick={onClose}
+                    >
+                      Back
+                    </button>
+                    <button
+                      className="px-4 py-2 bg-blue-500 text-white rounded hover:bg-blue-600"
+                      type="submit"
+                    >
+                      Review
+                    </button>
+                  </div>
+                </form>
+              ) : (
+                <div>
+                  <div className="mb-4">
+                    <label className="block text-gray-700" htmlFor="question1">
+                      Question 1
+                    </label>
+                    <p className="w-full px-3 py-2 border border-transparent rounded bg-white">{answerOne}</p>
+                  </div>
+                  <div className="mb-4">
+                    <label className="block text-gray-700" htmlFor="question2">
+                      Question 2
+                    </label>
+                    <p className="w-full px-3 py-2 border border-transparent rounded bg-white">{answerTwo}</p>
+                  </div>
+                  <div className="flex justify-end space-x-4">
+                    <button
+                      className="px-4 py-2 bg-gray-200 text-gray-700 rounded hover:bg-gray-300"
+                      type="button"
+                      onClick={() => setIsReviewMode(false)}
+                    >
+                      Back
+                    </button>
+                    <button
+                      className="px-4 py-2 bg-blue-500 text-white rounded hover:bg-blue-600"
+                      type="button"
+                      onClick={handleFinalSubmit}
+                    >
+                      Submit
+                    </button>
+                  </div>
+                </div>
+              )}
+            </>
+          ) : (
```

Comment:
Your render now has a nested ternary expression inside a ternary expression.

Can you better handle this with { isSubmissionComplete && isReviewMode ( ...code)} and some forms of it to manage the conditions.

Reply:
Moved the rendering logic into separate functions to avoid nested ternary expressions.